src/scf/mixing.jl

```diff
@@ -4,14 +4,30 @@ import Base: @kwdef
 
 # Mixing rules: (ρin, ρout) => ρnext, where ρout is produced by diagonalizing the
 # Hamiltonian at ρin These define the basic fix-point iteration, that are then combined with
-# acceleration methods (eg anderson).
+# acceleration methods (eg anderson). For the mixing interface we use `δF = ρout - ρin` and
+# `δρ = ρnext - ρin`, such that the mixing interface is `mix(mixing, basis, δF; kwargs...) -> δρ`
+# with the user being assumed to add this to ρin to get ρnext.
 # All these methods attempt to approximate the inverse Jacobian of the SCF step,
 # ``J^-1 = (1 - χ0 (vc + K_{xc}))^-1``, where vc is the Coulomb and ``K_{xc}`` the
 # exchange-correlation kernel. Note that "mixing" is sometimes used to refer to the combined
 # process of formulating the fixed-point and solving it; we call "mixing" only the first part
 # The notation in this file follows Herbst, Levitt arXiv:2009.01665
-#
-# The interface is `mix(m; basis, ρin, ρout, kwargs...) -> ρnext`
+
+@doc raw"""
+Simple mixing: ``J^{-1} ≈ α``
+"""
+@kwdef struct SimpleMixing
+    α::Real = 0.8
+end
+@timing "mixing Simple" function mix(mixing::SimpleMixing, ::PlaneWaveBasis, δFs...; kwargs...)
+    @assert length(δFs) > 0 && !isnothing(δFs[1])
```

Comment:
This assert feels unnecessary. I'm not a huge fan of the dFs... args instead of just passing an array, but we should discuss the API later anyway

Reply:
Otherwise LGTM!

Reply:
Me neither, actually. It's just to make it work. If you move to a more sensible datastructure the assert will go. It's just that I don't mess up (but it'll err anyway I suppose)